test/check-application

```diff
@@ -1201,6 +1208,13 @@ class TestApplication(testlib.MachineCase):
         b.click('.pf-c-modal-box button:contains(Restore)')
         b.wait(lambda: self.getContainerAttr("swamped-crate", "State") in 'Running')
 
+        # A new MAC address should have been generated
+        # Fixed in podman 4.4.0 https://github.com/containers/podman/issues/16666
+        import json
+        print(json.dumps(podman_version(self)))
```

Comment:
Debugging leftover. But to be sure that the version comparison works, you could do an `else:` below and check that the MAC address is equal.

Reply:
I actually think that's a neat idea! Let's test both cases.